apps/billing/payments/zibal_client.py

```
@@ -59,27 +61,27 @@ class ZibalError(Exception):
     RESULT_SUCCESS = 100
     RESULT_ALREADY_PAID = 201
     RESULT_PAYMENT_NOT_FOUND = 102
-    
+
     # Status codes
     STATUS_PAID = 1
     STATUS_PENDING = -1
     STATUS_CANCELLED = -2
-    
+
     def __init__(self):
```

Ruff (0.14.0)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)